CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 57.0.0-SNAPSHOT - unreleased
 
+### 🎁 New Features
+
+* Added `presetMenu` to the Admin `AlertBannerPanel` - enable the user to save preset banner message
+  configurations and load them later to be activated or edited.
```

Comment:
We usually reserve the backtick quoted names for Hoist components and APIs that we expect developers to know / interact with.  Here they are a bit of unnecessary detail - I would focus more on the functionality.  Something like:

> Enhanced the Admin alert banner feature with a new ability to save messages as presets. Useful for standardizing alert or downtime banners, where pre-approved language can be added as a preset and then easily loaded into a banner by members of an application support team."